code.js

```diff
@@ -1,3 +1,21 @@
 function fib(n) {
-    return n;
+
+    let array = [0,1];
+
+    if (n === 0) {
+
+        return [0];
+
+    } else if(n === 1) {
+
+        return array;
+
+    } else {
+        for (let i = 2; i <= n; i++) {
+
```

Comment:
Please write that in recursive, which means it should call the function itself.

Reply:
I just updated my code I think I did what I was supposed to do if not please let me know and I will change it!